README.rst

```diff
@@ -0,0 +1,6 @@
+Overview
+========
+This NApp implements Oplenflow multi tables
```

Comment:
Later on or on this PR could you also include the badges that we've also been using on other NApps similar to this https://raw.githubusercontent.com/kytos-ng/flow_manager/master/README.rst#L1-L8?

You can add `|Tag| |License| |Build| |Coverage| |Quality|`, and then once we have e2e tests you can also add the `|Stable|` OK?

Reply:
Adding just License, in the meantime.